### DISCUSSION TITLE
Battery Charge Limit hasn't been working

### DISCUSSION THREAD

**Discussion by StankovskiA** [seerge/g-helper#2337] [Q&A] [answered]
Hey, I wasn't sure where is the right place to ask about this, so i'll try here? 

I have the Asus TUF Dash F15 (bought it 1yr ago so probably the 2022 version) and a few months ago the everything on the G-Helper worked fine except the automatic refresh rate. The automatic refresh rate was blocking my laptop in some cases so I decided not to use it.

However, the battery charging limit has stopped working for at least 4-5 version now (not quite sure when) and im wondering if someone else is experiencing this?

Also sometimes I get bios battery erros on boot up, telling me to connect the charger to boot.

**Comment by seerge** [comment, marked as answer]
Hello,

> Also sometimes I get bios battery erros on boot up, telling me to connect the charger to boot.

If you get some "bios" (?) errors , most probably you have hardware issues with your device. Absolutely nothing has changed in the app in a terms of setting battery charge limit for last year or so.

Since you have completely ignored [**issue template and didn't provide any logs**](https://github.com/seerge/g-helper/issues) or clear reproduction scenario it's impossible to assist you further besides advising to read [**FAQ**](https://github.com/seerge/g-helper/wiki/FAQ) and trying general steps from [**Troubleshooting section**](https://github.com/seerge/g-helper/wiki/Troubleshooting)
